### PR TITLE
Two new features

### PR DESCRIPTION
**Pull request by suixin567**
1、npc script add a new ACT command : OPENBROWSER
aim:
user open a website with browser form npc
usage:
#ACT
OPENBROWSER www.google.com
2、the maps add a attributes : nobackcity . if checked ,players can not use back city scroll
3、add 5 language strings
4、fiexed AMain UI offset error.

### CODE REVIEW

**Comment by Pete107** [comment]
Is a whole class dedicated to finding the default browser necessary?
If I'm not mistaken Process.Start(url) will use the default browser..

File: Server/MirObjects/PlayerObject.cs [hunk 1]
```diff
@@ -12920,21 +12920,28 @@ private bool CanUseItem(UserItem item)
                         case 0:
                             if (CurrentMap.Info.NoEscape)
                             {
-                                ReceiveChat("You cannot use Dungeon Escapes here", ChatType.System);
+                                ReceiveChat(GameLanguage.CanNotDungeon, ChatType.System);
+                                return false;
+                            }
+                            break;
+                        case 1:
+                            if (CurrentMap.Info.NoEscape)
```
Comment (Suprcode) [inline Server/MirObjects/PlayerObject.cs]: this looks like a bug. It should be checking "BackCity"

**Comment by Suprcode** [comment]
Please can all references to "NoBackCity" be replaced with "NoTownTeleport" to stay consistent with its use.

**Comment by Suprcode** [comment]
> Is a whole class dedicated to finding the default browser necessary?
> If I'm not mistaken Process.Start(url) will use the default browser..

Yes that would open your native IE browser - however there is functionality in the class to look for chrome first and open chromes process with a default url

**Comment by suixin567** [comment]
@Suprcode I am  sorry  the bug is fixed.  and the name is changed too. 
and the feature about OpenbrowserHelper  class will not work right all the time ,because some  rogue software. but I think is enough now. 
so I hope a merger . And ,  I have written a another new feature make game server support Http service ,because  I need my website allow user login with the crystal game account, and so on... because there have no  DB like mysql 、sqlLite ... so , add a http service is convenient . what do you think?  if think so , I want to make a push .

**Comment by Suprcode** [comment]
> @Suprcode I am sorry the bug is fixed. and the name is changed too.
> and the feature about OpenbrowserHelper class will not work right all the time ,because some rogue software. but I think is enough now.
> so I hope a merger . And , I have written a another new feature make game server support Http service ,because I need my website allow user login with the crystal game account, and so on... because there have no DB like mysql 、sqlLite ... so , add a http service is convenient . what do you think? if think so , I want to make a push .

Sounds fine. Please add as a separate library if it requires several dependencies or classes though and ensure it is able to be turned off / configurable through an ini file / menu form.

**Pull request merged**